### PR TITLE
Allow robots, update URLs to apex domain, and rebuild sitemaps

### PR DESCRIPTION
**Pull request by jbranchaud**
Another PR to be merged after this app is living at egghead.io.

Open the SEO floodgates

### CODE REVIEW

File: public/robots.txt [hunk 1]
```diff
@@ -5,14 +5,12 @@ Allow: /
 Disallow: /discord
 Disallow: /_next
 Disallow: /purchase
-User-agent: *
-Disallow: /
```
Comment (jbranchaud) [inline public/robots.txt:9]: Allow robots to crawl the site

File: public/robots.txt [hunk 2]
```diff
@@ -5,14 +5,12 @@ Allow: /
...
-Sitemap: https://next.egghead.io/tags-sitemap-6.xml
+Host: https://egghead.io
+Sitemap: https://egghead.io/sitemap.xml
+Sitemap: https://egghead.io/tags-sitemap-0.xml
+Sitemap: https://egghead.io/tags-sitemap-1.xml
+Sitemap: https://egghead.io/tags-sitemap-2.xml
+Sitemap: https://egghead.io/tags-sitemap-3.xml
+Sitemap: https://egghead.io/tags-sitemap-4.xml
+Sitemap: https://egghead.io/tags-sitemap-5.xml
+Sitemap: https://egghead.io/tags-sitemap-6.xml
```
Comment (jbranchaud) [inline public/robots.txt:16]: Update the Host and Sitemap directives to use `egghead.io`

File: public/robots.txt (hunk 1 above)
Comment (joelhooks) [reply public/robots.txt:9]: this is actually generated!

File: next-sitemap.js [hunk 3]
```diff
@@ -31,6 +27,7 @@ module.exports = {
       'https://next.egghead.io/tags-sitemap-4.xml',
       'https://next.egghead.io/tags-sitemap-5.xml',
       'https://next.egghead.io/tags-sitemap-6.xml',
+      'http://egghead-sitemaps.s3.amazonaws.com/sitemaps/sitemap.xml.gz',
```
Comment (joelhooks) [inline next-sitemap.js:30]: egghead-rails will still generate the content sitemap, we should make sure it generates it with the appropriate URL

Comment (joelhooks) [reply next-sitemap.js:30]: https://github.com/eggheadio/egghead-rails/blob/8419d61081cc5305be69285c165213ddd0a87174/config%2Fsitemap.rb

verified that the root url is configured in the sitemap config

Comment (joelhooks) [reply next-sitemap.js:30]: plugin link for convenience: https://www.npmjs.com/package/next-sitemap#full-configuration-example

**Comment by joelhooks** [comment]
I did this in another PR

**Pull request closed without merging**